packages/cdk-routing-split/test/cases/unit/add.test.ts

```diff
@@ -0,0 +1,9 @@
+import * as when from '../../steps/when';
+
+describe('When an entity', async () => {
+    it('calls add', async () => {
```

Comment:
When running test:sandbox I get this error:
"Returning a Promise from "describe" is not supported. Tests must be defined synchronously."
Looks like async from describe should be removed. When I do that I get:
"If it should pass with deep equality, replace "toBe" with "toStrictEqual". When I replace toBe with toStrictEqual all tests are passing.

Reply:
Yea I thought about this all weekend 😆 Nathan made a big point about this before, it should be toStrictEqual(). I realized it early Friday when I was out and just never had a chance to fix it.

Reply:
oops i see ALSO - describe itself isn't supposed to have it, i missed that error before